site/content/en/docs/Guides/feature-stages.md

```diff
@@ -29,7 +29,7 @@ The current set of `alpha` and `beta` feature gates are:
 | Example Gate (not in use) | `Example` | Disabled | None | 0.13.0 |
 | [Player Tracking]({{< ref "/docs/Guides/player-tracking.md" >}}) | `PlayerTracking` | Disabled | `Alpha` | 1.6.0 |
 | [Custom resync period for FleetAutoscaler](https://github.com/googleforgames/agones/issues/1955) | `CustomFasSyncInterval` | Enabled | `Beta` | 1.25.0 |
-| [GameServer state filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239) | `StateAllocationFilter` | Disabled | `Alpha` | 1.14.0 |
+| [GameServer state filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239) | `StateAllocationFilter` | Enabled | `Beta` | 1.26.0 |
```

Comment:
Our documentation is updated on PR merge, so please see: https://agones.dev/site/docs/contribute/#within-a-page on how to only show this change on the new version of Agones.

You'll want to create two copies of the table, one with the old version in a `feature` shortcode that expires with 1.26.0 and one that publishes with `1.26.0`.

Reply:
I must admit that I erased them when I resolved conflicts 🤦
I'll fix it and other docs that follow the same pattern, Thanks.

Reply:
I added feature shortcode.